tests/test_tket.py

```diff
@@ -54,7 +54,7 @@ def _test_circuit(
         circuit_commands = [
             com for com in circuit.get_commands() if str(com.op) != "Barrier"
         ]
-        circuit_2 = qujax_args_to_tk(*tk_to_qujax_args(circuit))
+        circuit_2 = qujax_args_to_tk(*tk_to_qujax_args(circuit))  # type: ignore
```

Comment:
Why is this type ignore here required? Have you tried a cast? The return type of the function should be `circuit`?